langchain/sql_database.py

```diff
@@ -37,6 +37,7 @@ def __init__(
         metadata: Optional[MetaData] = None,
         ignore_tables: Optional[List[str]] = None,
         include_tables: Optional[List[str]] = None,
+        ignore_column_names_on_query: bool = True,
```

Comment:
marginal preference to have this opt in and default to False, eg `include_column_names_on_query = False`

Reply:
Done.